### PR TITLE
Base64 decoding problem

### PR DESCRIPTION
**Pull request by lolekthegreat**
files over 32K were not decoded properly

### CODE REVIEW

File: src/addons/jsonml.sql [hunk 1]
```diff
@@ -55,21 +55,6 @@ create or replace package body json_ml as
   begin
     if(jsonml_stylesheet is null) then 
     jsonml_stylesheet := xmltype('<?xml version="1.0" encoding="UTF-8"?>
-<!--
```
Comment (jsumners) [inline src/addons/jsonml.sql]: This license comment should be retained. Please add it back.

File: src/json_ext.sql [hunk 2]
```diff
@@ -647,67 +672,25 @@ create or replace package body json_ext as
   end base64;
 
 
-  function base64(l json_list) return blob as
+	function base64(l json_list) return blob as
```
Comment (jsumners) [inline src/json_ext.sql]: The indentation here is incorrect.

**Comment by jsumners** [comment]
I have added a couple inline comments. Please correct those changes.

**Comment by lolekthegreat** [comment]
I've made required changes.
Please rethink about the location of licence comment:
_ORA-03135: connection lost contact" is occuring during object deployment in database (ORACLE DB 11.2.0.4.0)_

**Comment by jsumners** [comment]
In order for the comment to be in the generated file it has to be part of the code that generates the file. There's really nowhere else to put it.

I will merge this soon. I have to decide how I will do it, though. Technically, the master branch (which you have targeted in this PR) is for releases only. But our contributing guidelines, which say to not target the master branch, are part of the upcoming v2.0.0 release. So I don't really feel like I can hold you to them.

**Comment by lolekthegreat** [comment]
Sorry for this. I just wanted to contribute and was not paying attention to
this. I will revert my changes on master branch and commit changes on
development.

Thx, Aleš

15. jul. 2016 14:30 je oseba "James Sumners" <notifications@github.com>
napisala:

**Comment by jsumners** [comment]
I very much welcome your contribution. If you are willing to close this PR and open a new one that targets the develop branch that would be awesome.

**Comment by lolekthegreat** [comment]
Closing this pull request. I'll made another pull request that targets the develop branch asap.

**Pull request closed without merging**